av_metrics/src/video/mod.rs

```diff
@@ -265,7 +267,7 @@ trait VideoMetric: Send + Sync {
             self.aggregate_frame_results(&out)
         } else {
             Err(MetricsError::VideoError {
-                reason: "Error processing the two videos".to_owned(),
+                reason: "Error processing the two videos",
```

Comment:
Here we should probably bubble up whatever error message came from our Result, instead of using a static string.

Reply:
Yep, I agree